spring-cloud-aws-samples/spring-cloud-aws-sqs-sample/src/main/resources/application.yaml

```diff
@@ -0,0 +1,8 @@
+cloud:
+    aws:
+        region:
+            static: eu-central-1
+        sqs:
```

Comment:
There should be no need to explicitly set `sqs.enabled` to true, right?

Reply:
True will fix it got left

Reply:
This is still waiting to be removed ;-)

Reply:
Ups :)